Tools/Scripts/webkitpy/benchmark_runner/run_benchmark.py

```diff
@@ -60,12 +60,13 @@ def config_argument_parser():
     parser.add_argument('--show-iteration-values', dest='show_iteration_values', action='store_true', help="Show the measured value for each iteration in addition to averages.")
     parser.add_argument('--generate-pgo-profiles', dest="generate_pgo_profiles", action='store_true', help="Collect LLVM profiles for PGO, and copy them to the diagnostics directory.")
     parser.add_argument('--profile', action='store_true', help="Collect profiling traces, and copy them to the diagnostic directory.")
-    parser.add_argument('browser_args', nargs='*', help='Additional arguments to pass to the browser process')
 
     group = parser.add_mutually_exclusive_group()
     group.add_argument('--browser-path', help='Specify the path to a non-default copy of the target browser as a path to the .app.')
     group.add_argument('--build-directory', dest='build_dir', help='Path to the browser executable (e.g. WebKitBuild/Release/).')
 
+    parser.add_argument('--browser-args', nargs=argparse.REMAINDER, help='Additional arguments to pass to the browser process. If using, this argument MUST be the last argument in the list (all sequental arguments or strings will be considered browser arguments).')
+
```

Comment:
This changes the CLI interface.

Is this change really needed? I would prefer if we can avoid breaking current hypothetical users of this script that are passing the browser args without the switch `--browser-args`  (note: I'm speculating, I don't know how everyone uses this script)

Reply:
You can pass arguments with dashes after two empty dashes. Try this:
```
Tools/Scripts/run-benchmark --plan sunspider --count 1 --browser chrome -- --arguments --for --chome
```
Then  `browser_args` should be `['--arguments', '--for', '--browser']`

This is not obvious so it is maybe a good idea to add some note about how to pass the arguments on the help text

Reply:
oh nice -- in this case I can leave the old API and update the description

Reply:
actually since this is not an urgent change I think I'll wait for a week or so and see if @emw-apple is ok with the change that adds the explicit `--browser-args`, out of a slight preference for me to have the explicit `--browser-args`. But if that positional argument is being used anywhere I'll change my PR to use the old API, like I sad above

Reply:
We are using the positional form internally. The reason I wrote it this way was that I felt it was more ergonomic than having to pass multiple arguments nested in a single argument, e.g. `run-benchmark --browser-args="--arguments --for --chrome"` because we'd then have to worry about nested levels of escaping.  If there's a compelling reason to switch to this approach that's okay with me—it'd be easy to adapt our internal usage—but I would prefer to keep it as is :)

> You can pass arguments with dashes after two empty dashes. Try this:
> 
> ```
> Tools/Scripts/run-benchmark --plan sunspider --count 1 --browser chrome -- --arguments --for --chrome
> ```
> 
> Then `browser_args` should be `['--arguments', '--for', '--chrome']`
> 
> This is not obvious so it is maybe a good idea to add some note about how to pass the arguments on the help text

+1, I should have done this earlier!

Reply:
reverted to use positional args! thanks for chimming in @emw-apple